README.md

```diff
@@ -574,19 +574,93 @@ Returns the preferred_terminology at the Terminology("self") level. <br> Expecte
 }
 ```
 
-#### POST
+#### PUT
 
-Create a reference to a prefered `Terminology` for the terminology (specified by id). <br> 
+Create or replace references to a prefered `Terminology` for the terminology (specified by id). <br> 
 Request body example:
  ```json
-[
-    {
-        "preferred_terminology": "tm--example3"
-    },
-    {
-        "preferred_terminology": "tm--example6"
-    }
-]
+{
+    "editor": "me",
+    "preferred_terminologies": [
+        {
+            "preferred_terminology": "tm--example1"
+        },
+        {
+            "preferred_terminology": "tm--example6"
+        }
+    ]
+}
+```
+
+#### DELETE
+Running the DELETE request will remove the preferred_terminology collection
+from the `Terminology` specified by the id
```

Comment:
The code above was added to the repo in an earlier PR. I don't know why it is showing them as additions in this PR. The green lines are the ones we want in the repo. Here is the PR https://github.com/NIH-NCPI/locutus/commit/7023691d7eac2cfef5d2727ffc5ff0acc1368d29